homeassistant/components/deconz/config_flow.py

```diff
@@ -141,22 +139,22 @@ def __init__(self):
 
     async def _create_entry(self):
         """Create entry for gateway."""
-        if CONF_BRIDGEID not in self.deconz_config:
+        if not self.bridgeid:
             session = aiohttp_client.async_get_clientsession(self.hass)
 
             try:
                 with async_timeout.timeout(10):
-                    gateway_config = await async_get_gateway_config(
+                    self.bridgeid = await async_get_bridge_id(
```

Comment:
SHould this be normalized ?

Reply:
Everything from the library itself exposes normalised ids since it also originates the normalisation function as well.